src/js/chromecast/ChromecastSessionManager.js

```diff
@@ -189,56 +183,58 @@ ChromecastSessionManager = Class.extend(/** @lends ChromecastSessionManager.prot
          }
          player.currentTime(currentTime || 0);
       });
-   },
+   }
 
    /**
     * @see https://developers.google.com/cast/docs/reference/chrome/cast.framework.CastContext
     * @returns {object} the current CastContext, if one exists
     */
-   getCastContext: getCastContext,
+
+   getCastContext() {
+      return cast.framework.CastContext.getInstance();
+   }
```

Comment:
What about doing this to a static function? Then we can use that in `isChromecastConnected`.